.github/CODEOWNERS

```diff
@@ -126,25 +125,21 @@
 
 
 # Core documentation
-# TODO(#12736): Restore code ownership to @BenHenning after 2021-05-24.
-/AUTHORS @seanlip
+/AUTHORS @seanlip @BenHenning
```

Comment:
Are we expecting to remove @seanlip from line 128 - 132. ([ref](https://github.com/oppia/oppia/pull/12737/files#diff-3d36a1bf06148bc6ba1ce2ed3d19de32ea708d955fed212c0d27c536f0bd4da7L123))

Reply:
Nope -- please keep me added. Per discussion with @BenHenning I will shadow his codeowners so that we have some redundancy.

Reply:
Sorry for the misunderstanding, I'm asking whether we need to add @BenHenning for /CHANGELOG, /CONTRIBUTORS, /LICENSE, /NOTICE? (as I see those [files were earlier assigned](https://github.com/oppia/oppia/pull/12737/files#diff-3d36a1bf06148bc6ba1ce2ed3d19de32ea708d955fed212c0d27c536f0bd4da7L123) to Ben!)

Reply:
Ah I see. I'm OK either way, actually -- let's just get this in!